### ISSUE TITLE
data question

### ISSUE THREAD

**Issue by fengcunsx** [marcpaga/esox#7]
Hello,

Using Nanopore to detect 8-oxoG is really cool, and as far as I know, you are the first to do this — it’s an inspiring and feasible approach.

I’m particularly interested in the detection algorithm. I have a question regarding your model training data:
Did you train the model using the batch1, batch2, batch3, and batch4 fast5 files together with the oligo__ref.fasta provided in your GitHub code repository?

Thank you for your time and for sharing your work.

**Comment by marcpaga** [comment]
Yes, the model is trained on all the oligos simultaneously. However, not all the data was used, since a small portion is kept for validation/testing. 

In the paper you can find the details in the methods section

> The oligo concatemers dataset (total of 2,460,266 reads) was first split into train and test sets. The test set consisted of 100 reads (chosen at random) per different oligo sequence (total of 22,000 reads). The train set consisted of all the reads (total 2,438,266 reads) not in the test set. To assess model performance during training, a portion of the train set was split into the validation set at random, and kept constant throughout the epochs (10% of the total training examples).

**Comment by fengcunsx** [comment]
Thank you very much for your reply!

I couldn’t find the reference FASTA file in PRJEB46810 — did you use only the oligo_ref.fasta from your GitHub code as the reference sequence to generate the training labels and data?

Thanks again for your clarification, and wishing you an endless streak of Nature publications!

**Comment by marcpaga** [comment]
Yes, you can basecall and align the basecalls to the reference to determine the underlying true sequence of each read.
